src/components/Snapshots.vue

```diff
@@ -349,6 +349,11 @@ interface IDownloadParameters {
   tiffCompression?: string;
 }
 
+function intFromString(value: string) {
```

Comment:
LGTM.
In my opinion, it would be great to have a math helper file with such functions, WDYT?

Reply:
After more thoughts, this function shouldn't be as is in a math helper file, because it makes the assumption that "not a number" should be treated as zero